universum/modules/output/terminal_based_output.py

```diff
@@ -4,6 +4,7 @@
 from .base_output import BaseOutput
 
 __all__ = [
+    "stdout",
```

Comment:
How about moving this function to 'BaseOutput', for example? It is used by two different but architecturally equal classes.